en/docs/reference/pricing.md

```diff
@@ -2,229 +2,145 @@
+- Managing the source code via a GitHub repository
 
-#### Scenario
-This scenario describes a hypothetical maintenance process carried out by airline maintenance. The airline uses Choreo to generate alerts by using webhooks when the aircraft maintenance status is updated. 
-The Aircraft Maintenance team carries out maintenance for each flight before take-off. A supervisor updates the maintenance status of the aircraft on a [Aircraft Maintenance Sheet](#AircraftMaintenanceSheet), which is a Google sheet. Upon updating the sheet, the [Aircraft Status Notifier](#aircraftNotifWebhook) webhook on Choreo gets triggered and updates the [Flight Scheduling Database](#flightSchedulingDatabase) that confirms if the aircraft is ready for take off,  whether it will be delayed due to maintenance, or whether it is not suitable for take off.
+- Continuous integrations (CI)
 
-#### Glossary
+- Continuous delivery (CD)
 
-- <a class="pricing" id="aircraftNotifWebhook">Aircraft Status Notifier</a>:A webhook in Choreo that listens to any events that trigger when the Aircraft maintenance sheet is updated.
-- <a class="pricing" id="AircraftMaintenanceSheet">Aircraft Maintenance Sheet</a>: A Google sheet that maintains the list of aircraft, their information, the maintenance schedule, and the maintenance status.
-- <a class="pricing" id="flightSchedulingDatabase">Flight Scheduling Database</a>:The database that includes details about  flights, the aircraft assigned to the flight route, and their maintenance state.
+- Deployments to cloud platforms are supported for the three plans as follows:
 
-<br/>
+   | **Free** | **Pay As You Go** | **Enterprise** |
+   |----------|-------------------|----------------|
+   | Azure    | Azure<br/>AWS<br/>GCP<br/>VMWare<br/>OpenShift<br/>Tanzu<br/>Any Kubernetes cluster |Azure<br/>AWS<br/>GCP<br/>VMWare<br/>OpenShift<br/>Tanzu<br/>Any Kubernetes cluster |
 
-#### Update maintenance status 
 
-![Update maintenance status](../assets/img/references/webhook-example1.png)
+#### Integration and automation
 
-Let’s see how the steps are calculated for the update maintenance process:
+Choreo supports the following integration and automation features for all the plans:
 
-- As illustrated above, the webhook registers with the Google sheet at the time the Webhook component is created on Choreo.
-- The supervisor initiates the update process by updating or adding the relevant information on the [Aircraft Maintenance Sheet](#AircraftMaintenanceSheet).
-- The Google Sheet triggers **1 event to the Webhook Listener,[Aircraft Status Notifier](#aircraftNotifWebhook)** on Choreo. This is counted as **1 step**.
-- The Webhook Listener,[Aircraft Status Notifier](#aircraftNotifWebhook) on Choreo then makes **1 database call to the [Flight Scheduling Database](#flightSchedulingDatabase)** to update the flight maintenance status. This is counted as 1 step. 
+- Triggers
```

Comment:
Are these components or features?

Reply:
In this context, they are features.

Reply:
It can be confusing, as we are referring to them as components, and when we come here it is features. Can give a wrong idea that Triggers and scheduled jobs are free